recipes/emscripten/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.37.27" %}
+
+package:
+  name: emscripten
+  version: {{ version }}
+
+source:
+  fn: emscripten-{{ version }}.tar.gz
+  url: https://github.com/kripken/emscripten/archive/{{ version }}.tar.gz
+  sha256: a345032415362a0a66e4886ecd751f6394237ff764b1f1c40dde25410792991c
+
+build:
+  number: 0
+  skip: true  # [win or py!=27]
```

Comment:
The code is not Python 3 compatible.

Also Windows dependencies require the latest compiler, which is incompatible with Python 2. So Windows is not supported at present.

Reply:
An attempt at Python 3 support in PR ( https://github.com/kripken/emscripten/pull/6983 ).